content/en/docs/concepts/storage/volumes.md

```diff
@@ -137,6 +139,8 @@ and the kubelet, set the `InTreePluginAWSUnregister` flag to `true`.
 
 ### azureDisk {#azuredisk}
 
+{{< feature-state for_k8s_version="v1.24" state="deprecated" >}}
```

Comment:
From #32901:
> CSI Migration for AzureDisk moved to GA in 1.24. CSI Migration for AzureFile moved to Beta on-by-default in 1.24. Both in-tree plugins were deprecated when CSI Migration moved to Beta off-by-default. So both in-tree plugins should be marked as deprecated in Kubernetes docs.

So, maybe..?
(I could not find the exact minor version in which this in-tree AzureDisk plugin was deprecated.
If you know, then please let us know!)

FYI: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#:~:text=1.18-,CSIMigrationAzureDisk,1.19,-1.22

```suggestion
### azureDisk (deprecated) {#azuredisk}

{{< feature-state for_k8s_version="v1.19" state="deprecated" >}}
```